apps/files/src/components/FilesAppBar.vue

```diff
@@ -228,12 +228,13 @@ export default {
           this.isLoadingSearch = false
         })
     },
-    focusFilenameFilter () {
-      this.$refs.filenameFilter.$el.querySelector('input').focus()
+    focusMobileSearchInput () {
+      this.$refs.mobileSearch.$el.querySelector('input').focus()
       // nested vuetify VList animation will block native autofocus, so we use this workaround...
+
       setTimeout(() => {
```

Comment:
You should be able to use nextTick here - https://vuejs.org/v2/api/#Vue-nextTick

Reply:
no this doesnt work for some reason :/

Reply:
Hmm, strange